### PR TITLE
Feature/gp2 932 dit link and responsive links

### PR DESCRIPTION
**Pull request by jakehopking**
To do (delete all that do not apply):

 - [X] CSS have been compiled.

### CODE REVIEW

File: src/scss/elements/_layout.scss [hunk 1]
```diff
@@ -5,6 +5,10 @@
 // LAYOUT
 // =======
 
+main#content {
```
Comment (dahfool) [inline src/scss/elements/_layout.scss:8]: This rule, might break some templates.

File: src/scss/elements/forms/_base.scss [hunk 2]
```diff
@@ -1,9 +1,9 @@
 @import 'general';
 @import 'validation';
-@import 'multiple-choice';
-@import 'multiple-select-auto-complete';
-@import 'radio-nested';
-@import 'text-input-with-button';
-@import 'typeahead';
+// @import 'multiple-choice';
```
Comment (dahfool) [inline src/scss/elements/forms/_base.scss:3]: Are we not using these files ?

File: src/scss/helpers/_colours.scss [hunk 3]
```diff
@@ -1,5 +1,5 @@
-@import "_design-tokens";
-@import "_design-tokens-temp";
+@import "../design-tokens";
+@import "../design-tokens-temp";
```
Comment (dahfool) [inline src/scss/helpers/_colours.scss:2]: Has the designs-tokens been moved ? This might break the script which import the tokens.

Comment (jakehopking) [reply src/scss/helpers/_colours.scss:2]: this is from when i was trying to update the great-cms imports... it failed on these imports. i'll revert.

File: src/scss/elements/_layout.scss (hunk 1 above)
Comment (jakehopking) [reply src/scss/elements/_layout.scss:8]: k - will remove

File: src/scss/elements/forms/_base.scss (hunk 2 above)
Comment (jakehopking) [reply src/scss/elements/forms/_base.scss:3]: I thought not after out conversation. I've add back, but omit 'multiple-choice' as I've re-implemented that using the checkbox version from great-styles.

**Review by dahfool — APPROVED** [review]

**Pull request merged**